### PR TITLE
Fix ws test 7.9.x : not authorized any close code

### PR DESCRIPTION
**Pull request by Nek-**
image

### CODE REVIEW

File: src/Rfc6455/MessageHandler/CloseFrameHandler.php [hunk 1]
```diff
@@ -27,10 +28,27 @@ public function process(Message $message, MessageProcessor $messageProcessor, Co
...
+        if (BitManipulation::frameSize($payload) > 1) {
+            $errorCode = BitManipulation::bytesFromTo($payload, 0, 1);
+
+            // https://tools.ietf.org/html/rfc6455#section-7.4
+            if (
+                (!in_array($errorCode, [
+                    Frame::CLOSE_NORMAL, Frame::CLOSE_GOING_AWAY, Frame::CLOSE_PROTOCOL_ERROR, Frame::CLOSE_GOING_AWAY,
+                    Frame::CLOSE_WRONG_DATA, Frame::CLOSE_INCOHERENT_DATA, Frame::CLOSE_TOO_BIG_TO_PROCESS,
+                    Frame::CLOSE_POLICY_VIOLATION, Frame::CLOSE_MISSING_EXTENSION, Frame::CLOSE_UNEXPECTING_CONDITION]
+                ) && $errorCode < 3000 && $errorCode > 900) || $errorCode > 4999 || $errorCode < 1000
```
Comment (folliked) [inline src/Rfc6455/MessageHandler/CloseFrameHandler.php]: need refactoring

File: tests/Woketo/Rfc6455/MessageHandler/CloseFrameHandlerTest.php [hunk 2]
```diff
@@ -64,4 +64,24 @@ public function testItCloseWithProtocolErrorWhenFrameIsNotValid()
...
+        $socket->end()->shouldBeCalled();
+
+        // Normal close frame without mask
+        $message = new Message();
+        $message->addFrame(new Frame(BitManipulation::hexArrayToString(['88', '02', '00', 'F7'])));
+
+        $handler = new CloseFrameHandler();
+        $this->assertTrue($handler->supports($message));
+        $handler->process($message, $messageProcessor->reveal(), $socket->reveal());
+    }
```
Comment (folliked) [inline tests/Woketo/Rfc6455/MessageHandler/CloseFrameHandlerTest.php]: It does not test all the use case of the new code.

Comment (Nek-) [inline tests/Woketo/Rfc6455/MessageHandler/CloseFrameHandlerTest.php]: You really think testing all possibilities of failure in that case is relevant ?

File: src/Rfc6455/MessageHandler/CloseFrameHandler.php (hunk 1 above)
Comment (folliked) [inline src/Rfc6455/MessageHandler/CloseFrameHandler.php]: ok

Comment (Nek-) [inline src/Rfc6455/MessageHandler/CloseFrameHandler.php]: You can milestone 1.1 !

**Pull request merged**